build.gradle

```diff
@@ -20,20 +20,12 @@ import org.springframework.boot.gradle.plugin.SpringBootPlugin
 
 // ./gradlew clean && ./gradlew uploadArchives -Prelease
 
-buildscript {
-    repositories {
-        mavenCentral()
-        maven { url "https://plugins.gradle.org/m2/" }
-        maven { url "https://repo.spring.io/plugins-release" }
-    }
-    dependencies {
-        classpath "com.puppycrawl.tools:checkstyle:10.3.1"
```

Comment:
checkstyle is omitted.
Is this on purpose?

Reply:
The checkstyle plugin is a part of the gradle core plugins. There's no reason to list here.
https://docs.gradle.org/current/userguide/checkstyle_plugin.html